SonarQube.TeamBuild.PostProcessor/Resources.resx

```diff
@@ -125,7 +125,7 @@ Analysis config file: {2}
 Please delete the analysis config file and try the build again.</value>
   </data>
   <data name="ERROR_CmdLine_DisallowedArgument" xml:space="preserve">
-    <value>This command line argument is not valid in the "end" phase: {0}</value>
+    <value>This setting is not valid in the "end" phase: {0}. Possible cause: you are not running the latest C# plugin version.</value>
```

Comment:
Done. I changed it to "This setting is not valid the "end" phase in this version of the C# plugin: {0}" as task customer are not aware of the Runner, and all other customers will still need to update their C# plugin to get a newer version of the post processor